javascript/grammar.json

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
I think there are a few entries that are missing here:

null_literal for null
boolean_literals for true/false
decimal_numeric_literals
hexadecimal_numeric_literals
including the hexadecimal_escape_sequences
unicode_escape_sequences
array_literals
string_literals
regular_expression_literals

All with 'true' compat values.